tests/robot-tests/tests/admin_and_public/bau/publish_release_and_amend_2.robot

```diff
@@ -8,25 +8,89 @@ Suite Setup       user signs in as bau1
 Suite Teardown    user closes the browser
 
 *** Variables ***
-${DETAILS_HEADING}  Academic Year 2020/21
+${DETAILS_HEADING}  Academic Year Q1 2020/21
```

Comment:
`DETAILS_HEADING` is way too generic of a name (we use `details` components . It sounds like this should be `RELEASE_NAME` instead